index.html

```diff
@@ -82,6 +82,7 @@
         openmct.install(openmct.plugins.Filters(['table', 'telemetry.plot.overlay']));
         openmct.install(openmct.plugins.ObjectMigration());
         openmct.install(openmct.plugins.ClearData(['table', 'telemetry.plot.overlay', 'telemetry.plot.stacked']));
+        openmct.install(openmct.plugins.ConditionalStyles(['layout']));
```

Comment:
This is the approach we've used elsewhere for selectively applying functionality in plugins, but it's one I'd like to move away from. Perhaps the style inspector could be applicable to objects that have a `.styles` property?